junit-engine-api/src/main/java/org/junit/gen5/engine/ExecutionRequest.java

```diff
@@ -25,12 +25,16 @@
 
 	private final EngineExecutionListener engineExecutionListener;
 
+	private ConfigurationParameters configurationParameters;
+
 	private final Map<String, Object> attributes = new HashMap<>();
```

Comment:
And off they go... 😄 

I removed them and declared the variable final. I will merge this PR later this day.